src/libraries/System.Runtime.Caching/Directory.Build.targets

```diff
@@ -1,9 +1,8 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.targets" />
   <PropertyGroup>
-    <!-- this assembly is inbox in desktop, do not version it unless you
-         plan on shipping a new desktop version out of band. Instead add API
-         to a different assembly. -->
+    <!-- This assembly is inbox in .NETFramework, ensure that the .NETStandard assembly
+         remains <= the desktop version -->
```

Comment:
Tiny nit: I see desktop was changed to .NETFramework in line 4, but not in line 5. Not asking to retrigger the CI just for this, but maybe the same language should be applied here.

Reply:
I'd prefer to get it right.